src/components/InstanceFilters/InstanceFilters.js

```diff
@@ -43,11 +45,127 @@ const InstanceFilters = props => {
       modesOfIssuance,
       natureOfContentTerms,
       tagsRecords,
+      query: { query = '', filters = '' },
+      onFetchFacets,
+      parentResources,
     },
     onChange,
     onClear,
   } = props;
 
+  const {
+    facets: { records = {} },
+  } = parentResources;
+
+  const [accordions, setAccordions] = useState({
+    effectiveLocation: true,
```

Comment:
Please discuss with Magda whether do need facets to be opened by default

Reply:
Magda said that it doesn't matter, I left It as it is.